src/components/calcite-popover/calcite-popover.e2e.ts

```diff
@@ -0,0 +1,59 @@
+import { newE2EPage } from "@stencil/core/testing";
+
+import { hidden, renders } from "../../tests/commonTests";
+
+describe("calcite-popover", () => {
+  it("renders", async () => renders("calcite-popover"));
+
+  it("honors hidden attribute", async () => hidden("calcite-popover"));
+
+  it("popover defaults", async () => {
+    const page = await newE2EPage();
+
+    await page.setContent("<calcite-popover></calcite-popover>");
+
+    const element = await page.find("calcite-popover");
```

Comment:
The `defaults` test helper could simplify this test. See https://github.com/Esri/calcite-app-components/blob/master/src/components/calcite-block/calcite-block.e2e.ts#L10-L20 as an example.